### PR TITLE
ROE-101 Add Individual Beneficial Owner Page

### PR DESCRIPTION
**Pull request by mwestacott**
### https://companieshouse.atlassian.net/browse/ROE-40

* New page added with appropriate URLs, routing and controller
* Minor refactoring of config index module to benefit from reuse
of defined constant names

### Work checklist

- [X] Tests added where applicable
- [X] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -18,24 +18,26 @@ export const COOKIE_DOMAIN = getEnvironmentValue("COOKIE_DOMAIN");
...
-
-// TEMPLATE PATH
+// TEMPLATEs
 export const BENEFICIAL_OWNER_OTHER_PAGE = "beneficial-owner-other";
 export const BENEFICIAL_OWNER_TYPE_PAGE = "beneficial-owner-type";
 export const ENTITY_PAGE = "entity";
 export const LANDING_PAGE = "landing";
 export const MANAGING_OFFICER_PAGE = "managing-officer";
 export const PRESENTER_PAGE = "presenter";
+export const BENEFICIAL_OWNER_INDIVIDUAL_PAGE = "beneficial-owner-individual";
```
Comment (mattch1) [inline src/config/index.ts:28]: I think we have decided to arranged these alphabetically.

**Comment by mattch1** [comment]
All good, just one minor thing that came up on another PR.

File: src/config/index.ts (hunk 1 above)
Comment (mwestacott) [reply src/config/index.ts:28]: 👍  will change.

**Comment by markpit** [comment]
looks good, no other comments to add

**Pull request merged**